Dockerfile

```diff
@@ -13,18 +38,16 @@ ENV farmer_port="null"
 ENV testnet="false"
 ENV TZ="UTC"
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y bc curl lsb-release python3 tar bash ca-certificates git openssl unzip wget python3-pip sudo acl build-essential python3-dev python3.8-venv python3.8-distutils python-is-python3 vim tzdata && \
-    rm -rf /var/lib/apt/lists/* && \
-    ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone && \
-    dpkg-reconfigure -f noninteractive tzdata
-
-ARG BRANCH=latest
+COPY --from=install /chia-blockchain /chia-blockchain
 
-RUN echo "cloning ${BRANCH}" && \
-    git clone --branch ${BRANCH} https://github.com/Chia-Network/chia-blockchain.git && \
-    cd chia-blockchain && \
-    git submodule update --init mozilla-ca && \
-    /usr/bin/sh ./install.sh
+RUN chmod 1777 /tmp && \
+    apt-get update -q && \
+    python_version=$(ls /chia-blockchain/venv/lib) && \
```

Comment:
Is this actually necessary? can we just install `python3-distutils` and `python3-venv` instead or trying to get a specific python 3 version?